src/audio/dai.c

```diff
@@ -393,7 +393,9 @@ static void dai_data_config(struct comp_dev *dev)
 		dd->stream_id = dd->dai_config->alh.stream_id;
 		break;
 	case SOF_DAI_IMX_SAI:
-		COMPILER_FALLTHROUGH;
+		dd->config.burst_elems =
+			dd->dai->plat_data.fifo[dai->direction].depth;
+		//COMPILER_FALLTHROUGH;
```

Comment:
This change doesn't look good at all. You're falling through and running the same statement twice. What's the point in the first place?